tools/cluster.py

```diff
@@ -342,9 +342,17 @@ def destroy(cluster_name, delete_groups=False, region=default_region):
         log.info('Terminating master...')
         for i in masters:
             i.terminate()
+
         log.info('Terminating slaves...')
         for i in slaves:
             i.terminate()
+
+        while wait_termination and all_instances:
```

Comment:
Would be good if we have:
- logging 'Waiting termination...'
- timeout (we could potentially wait forever)